app-shells/loksh/loksh-7.0.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson
+
+DESCRIPTION="Linux port of OpenBSD's ksh"
+HOMEPAGE="https://github.com/dimkr/loksh"
+SRC_URI="https://github.com/dimkr/loksh/releases/download/${PV}/${P}.tar.xz"
+
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
+
+DEPEND="sys-libs/ncurses"
```

Comment:
Need to keep the := if we're linking against it.

Also, avoid merge commits -- no need for them in pull requests, just rebase.

Reply:
Should I revert?
Sorry, my first time contributing to gentoo so things are a bit messy.

Reply:
No worries! See [here](https://devmanual.gentoo.org/ebuild-maintenance/git/index.html#configuration-specific-to-the-gentoo-repository) to avoid merge commits (you can adjust your config to not do merges).

I don't mind trying to detangle/drop the merge commit this time around though. Or you can have a play and see if you can get it done. You might just be able to drop it with 'git rebase -i'.